src/test/resources/org/broadinstitute/hellbender/engine/example_variants_WithReads.vcf

```diff
@@ -0,0 +1,8 @@
+##fileformat=VCFv4.1
```

Comment:
These new test files should probably be named after the class in whose tests they are used.